Source/ISHRemote/Trisoft.ISHRemote/Objects/FieldElements.cs

```diff
@@ -974,5 +974,13 @@ internal class FieldElements
         public const string AnnotationSnapshotId = "FISHANNOTSNAPSHOTID";
 
         #endregion
+
+        #region BackgroundTask Fields
+        
+        /// <summary>
+        /// The ProgressId of the BackgroundTask.
+        /// </summary>
+        public const string ProgressId = "PROGRESSID";
```

Comment:
Prefer `BackgroundTaskProgressId` instead of `ProgressId` ... see higher in this file like Annotation or TranslationJob...

Reply:
Done